common/men_at_arms_types/temp_draft_doc.txt

```diff
@@ -0,0 +1,587 @@
+	low_maintenance_cost = { gold = cost_180_low_maint_cost }
+	high_maintenance_cost = { gold = cost_180_high_maint_cost }
+	
+	stack = 50
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = kunchong
+}
+
+#Ogre
+mauler = {
+	type = heavy_infantry
+	
+	damage = 37
+	toughness = 40
+	pursuit = 0
+	screen = 20
+
+	counters = {
+		skirmishers = 1
+		pikemen = 1
+	}
+	
+	buy_cost = { gold = cost_108_recruitment_cost }
+	low_maintenance_cost = { gold = cost_108_low_maint_cost }
+	high_maintenance_cost = { gold = cost_108_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = mauler
+}
+
+#Pandaren
+mushan_rider = {
+	type = heavy_cavalry
+	
+	damage = 185
+	toughness = 150
+	pursuit = 60
+	screen = 25
+	
+	terrain_bonus = {
+		plains = { damage = 20 toughness = 15 pursuit = 10 }
+		drylands = { damage = 20 toughness = 15 pursuit = 10 }
+		desert = { damage = 20 toughness = 15 pursuit = 10 }
+		hills = { damage = -5 }
+		jungle = { damage = 5 }
+		mountains = { damage = -10 }
+		desert_mountains = { damage = -10 }
+		wetlands = { damage = -10 }
+	}
+
+	counters = {
+		archers = 2
+		caster = 1
+	}
+
+	buy_cost = { gold = cost_240_recruitment_cost }
+	low_maintenance_cost = { gold = cost_240_low_maint_cost }
+	high_maintenance_cost = { gold = cost_240_high_maint_cost }
+	
+	stack = 50
+	ai_quality = { value = culture_ai_weight_mushan }
+	icon = mushan
+}
+#Qiriaji
+silithid_colossus = { 
+	type = massive_infantry
+	
+	damage = 135
+	toughness = 125
+	pursuit = 15
+	screen = 20
+	
+	terrain_bonus = {
+		desert = { damage = 10 toughness = 10 pursuit = 15 }
+		desert_mountains = { damage = 10 toughness = 20 pursuit = 10 }
+	}
+
+	counters = {
+		skimishers = 3
+	}
+
+	buy_cost = { gold = cost_180_recruitment_cost }
+	low_maintenance_cost = { gold = cost_180_low_maint_cost }
+	high_maintenance_cost = { gold = cost_180_high_maint_cost }
+	
+	stack = 50
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = silithid_colossus
+}
+#Quillboar
+thornweaver = {
+	type = caster
+	
+	damage = 45
+	toughness = 25
+	pursuit = 10
+	screen = 10
+	
+	terrain_bonus = {
+		desert = { damage = 25 toughness = 25 pursuit = 10 }
+		desert_mountains = { damage = 25 toughness = 25 pursuit = 10 }
+		hills = { damage = 25 toughness = 25 pursuit = 10 }
+		drylands = { damage = 25 toughness = 25 pursuit = 10 }
+	}
+
+	counters = {
+		pikemen = 1
+		massive_infantry = 1
+		heavy_infantry = 1
+	}
+
+	buy_cost = { gold = cost_120_recruitment_cost }
+	low_maintenance_cost = { gold = cost_120_low_maint_cost }
+	high_maintenance_cost = { gold = cost_120_high_maint_cost }
+	
+	stack = 75
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = thornweaver
+}
+#Titanforged
+iron_giant = {
+	type = massive_infantry
+	
+	damage = 125
+	toughness = 136
+	pursuit = 5
+	screen = 50
+	
+	terrain_bonus = {
+		mountains = { damage = 15 toughness = 10 screen = 10}
+	}
+
+	counters = {
+		skirmishers = 2
+		light_aerial = 2
```

Comment:
Light aerial or archers? Google doc says archers and I can't see why they would be good vs air.